securedrop/sass/_base.sass

```diff
@@ -151,6 +151,9 @@
         padding: 5px
         margin: 5px
 
+        &:dir(rtl)
+          float: left
+
```

Comment:
For maintainability would you mind moving this up under the **float: right** ?

Reply:
Sure - modified :)